ocp_resources/resource.py

```diff
@@ -319,6 +319,7 @@ class ApiGroup:
         UPLOAD_CDI_KUBEVIRT_IO = "upload.cdi.kubevirt.io"
         V2V_KUBEVIRT_IO = "v2v.kubevirt.io"
         VM_KUBEVIRT_IO = "vm.kubevirt.io"
+        INFRA_FEATURES = "operators.openshift.io/infrastructure-features"
```

Comment:
this is not match our code, you should add only "operators.openshift.io" here
name should end with _IO and should be placed in the right line (a, b, c ....)